### PR TITLE
Adds updated Mlops/stage1/get-started-bq notebook to the community folder

### PR DESCRIPTION
**Pull request by krishr2d2**
Opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb [hunk 1]
```diff
@@ -1144,87 +1109,37 @@
...
-        "        print(e)\n",
+        "# Delete the dataset using the Vertex dataset object\n",
+        "dataset.delete()\n",
         "\n",
-        "    if \"BUCKET_NAME\" in globals():\n",
-        "        ! gsutil rm -r $BUCKET_NAME"
+        "if os.getenv(\"IS_TESTING\"):\n",
+        "    # Delete the created GCS bucket\n",
+        "    ! gsutil rm -r $BUCKET_URI\n",
+        "    # Delete the created BigQuery datasets (move up ?? ask Karl)\n",
```
Comment (kweinmeister) [inline notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb:1140]: @krishr2d2 I think this is in a good spot (remember to delete the comment about moving up

File: notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb [hunk 2]
```diff
@@ -1081,9 +1046,9 @@
         "\n",
         "Currently, there is no direct data feeding connector between BigQuery and the open source XGBoost.\n",
         "\n",
-        "The BigQuery ML service has XGBoost training builtin.\n",
+        "The BigQuery ML service has builtin XGBoost training module.\n",
```
Comment (kweinmeister) [inline notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb:1049]: has a built-in

File: notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb [hunk 3]
```diff
@@ -8,7 +8,7 @@
       },
```
Comment (andrewferlitsch) [inline notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb:8]: <p>Add open in Colab</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/392/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb:8]: <p>you will only -&gt; you only</p><p> </p><p>Note: TW rules - no future tense</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/392/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb:8]: <p>drop will - no future tense</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/392/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (andrewferlitsch) [inline notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb:8]: <p>reduce installs to only the packages used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/392/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

File: notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb (hunk 1 above)
Comment (krishr2d2) [reply notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb]: Done.

File: notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb (hunk 2 above)
Comment (krishr2d2) [reply notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb]: changed builtin to built-in everywhere.

File: notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb (hunk 3 above)
Comment (krishr2d2) [reply notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb:8]: removed outstanding future tenses.

Comment (krishr2d2) [reply notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb:8]: Done.

Comment (krishr2d2) [reply notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb:8]: Done.

Comment (krishr2d2) [reply notebooks/community/ml_ops/stage1/get_started_bq_datasets.ipynb:8]: This one is in Mlops folder. Is it ok to add Colab link ?

**Pull request merged**